angular/projects/admin/src/index.html

```diff
@@ -10,6 +10,7 @@
     <link rel="icon"
           type="image/x-icon"
           href="/favicon.ico">
+    <script src="https://cdn.ckeditor.com/4.6.1/full/ckeditor.js"></script>
```

Comment:
Ouchie, do we need to add it as a global dependency on main document? :-(

Reply:
Yup, we need to add that based on [installation](https://github.com/chymz/ng2-ckeditor) step 

But we can avoid that https://github.com/chymz/ng2-ckeditor/issues/228

Reply:
I've tried the solution, but `"allowOutsideOutDir"` for Angular 8 version